### PR TITLE
GH #180

### PR DESCRIPTION
**Pull request by vpeil**

### CODE REVIEW

**Comment by vpeil** [comment]
to check if R cmd syntax is correct... (I added what some googling offered to me)

File: fabfile.py [hunk 1]
```diff
@@ -2,25 +2,29 @@
...
         sys.exit("Input file and output file are required.")
 
     put("bin/fetch.pl", "openapc/fetch.pl")
-    put("data/" + input_file, "openapc/" + input_file)
+    in_file = os.path.basename(input)
+    put(input, "openapc/" + in_file)
 
     with cd("openapc/"):
-        run("perl fetch.pl --input " + input_file + " --output tmp_ut.csv")
+        print(blue("Running script!"))
```
Comment (cbroschinski) [inline fabfile.py]: 'blue' is not in namesapce

File: fabfile.py [hunk 2]
```diff
@@ -2,25 +2,29 @@
...
+        run("perl fetch.pl --input " + in_file + " --output tmp_ut.csv")
         get("tmp_ut.csv", "tmp_ut.csv")
-        local("R CMD BATCH R/isi_add.r tmp.txt " + output_file)
+        local("R CMD BATCH '--args tmp.txt " + output +"' R/isi_add.r")
```
Comment (cbroschinski) [inline fabfile.py]: Does not work yet:
`[localhost] local: R CMD BATCH '--args tmp.txt python/out_ut.csv' R/isi_add.r`
`Fatal error: local() encountered an error (return code 1) while executing 'R CMD BATCH '--args tmp.txt python/out_ut.csv' R/isi_add.r'`

**Comment by vpeil** [comment]
aargh, some leftover from debugging! please delete this line!


Am 04.11.2016 um 11:22 schrieb Christoph Broschinski:

**Pull request merged**